smack-openpgp/src/main/java/org/jivesoftware/smackx/ox/OpenPgpSecretKeyBackupPassphrase.java

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ * Copyright 2020 Paul Schaub.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.ox;
+
+import java.util.regex.Pattern;
+
+/**
+ * Represents a secret key backup passphrase whose format is described in XEP-0373 §5.3.
+ *
+ * @see <a href="https://xmpp.org/extensions/xep-0373.html#backup-encryption">
+ *      XEP-0373 §5.4 Encrypting the Secret Key Backup</a>
+ */
+public class OpenPgpSecretKeyBackupPassphrase implements CharSequence {
+
+    private static final Pattern PASSPHRASE_PATTERN = Pattern.compile(
+            "^([123456789ABCDEFGHIJKLMNPQRSTUVWXYZ]{4}-){5}[123456789ABCDEFGHIJKLMNPQRSTUVWXYZ]{4}$");
+
+    private final String passphrase;
+
+    public OpenPgpSecretKeyBackupPassphrase(String passphrase) {
+        if (!PASSPHRASE_PATTERN.matcher(passphrase).matches()) {
```

Comment:
I'd love to see unit tests for this constructor. One with a valid passphrase and one with an invalid one would be a great start. :)

Reply:
There is a test for a valid construction already, but I happily add another one as well :)

Reply:
Added dedicated junit tests.